crypto/hmac/hm_ameth.c

```diff
@@ -47,7 +47,8 @@ static int hmac_pkey_ctrl(EVP_PKEY *pkey, int op, long arg1, void *arg2)
 
 static int hmac_pkey_public_cmp(const EVP_PKEY *a, const EVP_PKEY *b)
```

Comment:
use hmac_pkey_public_eq ??

Reply:
This is 1.1.1 branch. I do not think we want such kind of changes there.